.gitignore

```diff
@@ -76,3 +76,8 @@ output.map
 /Common/Utils/test_tools/CMakeSettings.json
 /cov-int/
 /Scripts/kobo-build-rootfs/build-config-user.sh
+<<<<<<< Updated upstream
+=======
+android-studio/build.gradle
+CMakeLists.txt
+>>>>>>> Stashed changes
```

Comment:
this is merge conflict and must be fixed.

PS : build.gradle and CMakelists.txt are part of project source and should not be added to  .gitignore